acceptance/tests/environment/use_enc_environment_for_files.rb

```diff
@@ -1,9 +1,9 @@
 test_name "Agent should use environment given by ENC for fetching remote files"
 
-testdir = master.tmpdir('respect_enc_test')
+testdir = create_tmpdir_for_user master, 'respect_enc_test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.